devices/waxman.js

```diff
@@ -19,4 +20,26 @@ module.exports = [
             await reporting.temperature(endpoint);
         },
     },
+    {
+        zigbeeModel: ['House Water Valve - MDL-TBD'],
+        // Should work with all manufacturer model numbers for the 2.0 series:
+        // 8850000 3/4"
+        // 8850100 1"
+        // 8850200 1-1/4"
+        // 8850300 1-1/2"
+        // 8850310 2"
+        model: '8850100',
```

Comment:
Given the above comment, I'm not sure if this is the right way to "identify" this model. Open to whatever the existing pattern is here, if any.

Reply:
This is fine!